crates/evm/src/evm/system_events.rs

```diff
@@ -1,16 +1,11 @@
+use alloy_consensus::TxEip1559;
+use alloy_primitives::{address, Address, TxKind, U256};
 use reth_primitives::{
-    address, Address, Signature, Transaction, TransactionSigned, TransactionSignedEcRecovered,
-    TransactionSignedNoHash, TxEip1559, TxKind, U256,
+    Signature, Transaction, TransactionSigned, TransactionSignedEcRecovered,
+    TransactionSignedNoHash,
 };
 
-use super::system_contracts::{BitcoinLightClient, Bridge};
-
-/// This is a special signature to force tx.signer to be set to SYSTEM_SIGNER
-pub const SYSTEM_SIGNATURE: Signature = Signature {
```

Comment:
https://github.com/alloy-rs/core/pull/810

Reply:
ohh nice one ignore my previous comments about fn not being const :D

Reply:
I guess we still gotta wait for a release